### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by jondlm**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [ ] Edge (Win 10)
- ~~Unit tests written (`common` at minimum)~~
- [x] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [ ] One core team UX approval

image

### CODE REVIEW

File: gulp/docs.js [hunk 1]
```diff
@@ -45,20 +45,31 @@ module.exports = {
...
-								if (property.key.name === 'render') {
-									definition = path;
-								}
-							});
-							return false;
-						}
-					});
-					return definition;
-				});
+				var docs = reactDocgen.parse(
```
Comment (jondlm) [inline gulp/docs.js]: No functionality changes here, just laying the groundwork for future improvements

File: src/docs/index.jsx [hunk 2]
```diff
@@ -237,7 +235,16 @@ const Component = React.createClass({
 								<tr key={propName}>
 									<td>{propName}</td>
 									<td><PropType type={propDetails.type} componentName={componentName} /></td>
-									<td>{String(propDetails.required)}</td>
+									<td>{propDetails.required ? 'yes' : 'no'}</td>
```
Comment (jondlm) [inline src/docs/index.jsx]: Switched to `yes/no` instead of `true/false`

File: gulp/docs.js (hunk 1 above)
Comment (ogupte) [inline gulp/docs.js]: yes parsing for the child props is one thing i'd like to add to this AST hook

Comment (jondlm) [inline gulp/docs.js]: Totally, that's what I was thinking. I started going down that rabbit hole but decided to save it for another day!

**Pull request merged**